### PR TITLE
Rocm3.8 doc update

### PR DESCRIPTION
**Pull request by sunway513**
Updates on build from source instructions and whl package releases.

### CODE REVIEW

File: rocm_docs/tensorflow-install-basic.md [hunk 1]
```diff
@@ -61,12 +61,11 @@ Install ROCm pkgs:
...
 sudo apt-get update && \
     sudo apt-get install -y --allow-unauthenticated \
-    rocm-dkms rocm-dev rocm-libs hipcub rccl \
+    rocm-dkms rocm-dev rocm-libs rccl \
     rocm-device-libs \
     hsa-ext-rocr-dev hsakmt-roct-dev hsa-rocr-dev \
     rocm-opencl rocm-opencl-dev \
-    rocm-utils \
-    miopen-hip miopengemm
+    rocm-utils
```
Comment (deven-amd) [inline rocm_docs/tensorflow-install-basic.md:68]: removing `hipcub`, `miopen-hip` and `miopengemm` from the install list...is that by design?

**Review by deven-amd — APPROVED** [review]
LGTM

**Comment by deven-amd** [comment]
jenkins : retest rocm-xla please

File: rocm_docs/tensorflow-install-basic.md (hunk 1 above)
Comment (deven-amd) [reply rocm_docs/tensorflow-install-basic.md:68]: checked with Peng...the changes are intended as those packages are now part of the `rocm-libs` meta-package

**Review by mvermeulen — APPROVED** [review]
LGTM

**Pull request merged**